app/Services/Reports/Volumes/ImageAnnotations/AbundanceReportGenerator.php

```diff
@@ -133,6 +222,17 @@ protected function createCsv($rows, $title, $labels)
             $csv->putCsv($row);
         }
 
+        $labelsCount = $labels->count();
+        $emptyImagesQuery->orderBy('filename')->chunk(1000, function ($images) use ($labelsCount, $csv) {
+            foreach ($images as $image) {
+                $row = [$image->filename];
+                for ($i = 0; $i < $labelsCount; $i++) {
+                    $row[] = 0;
+                }
+                $csv->putCsv($row);
+            }
+        });
```

Comment:
Using a [lazy stream](https://laravel.com/docs/11.x/queries#streaming-results-lazily) could make this slightly more readable:

```suggestion
        foreach ($emptyImagesQuery->orderBy('filename')->lazy() as $image) {
            $row = [$image->filename];
            for ($i = 0; $i < $labelsCount; $i++) {
                $row[] = 0;
            }
            $csv->putCsv($row);
        }
```